src/site/content/en/react/react.11tydata.js

```diff
@@ -7,46 +7,27 @@ module.exports = {
     slug: 'react',
```

Comment:
Not sure if it's a big deal but wanted to mention that the last article in the Next.js topic will have buttons at the bottom which flow into the Create React App topic.

![image](https://user-images.githubusercontent.com/1066253/66943494-213fb400-f000-11e9-9ad7-6a1f708cc327.png)

Reply:
That is awkward but not a dealbreaker.